libr/bin/bfile.c

```diff
@@ -7,12 +7,21 @@
 #define R_STRING_SCAN_BUFFER_SIZE 2048
 #define R_STRING_MAX_UNI_BLOCKS 4
 
-static void print_string(RBinString *string, RBinFile *bf) {
-	if (!string || !bf) {
-		return;
+#define K(x) sdb_fmt("%"PFMT64x, x)
+
+static RBinString *find_string_at (RBinFile *bf, RList *ret, ut64 addr) {
+	if (addr != 0 && addr != UT64_MAX) {
```

Comment:
why this check? i think you can just avoid it and look it up in the hashtable... if there is a value for those keys, we should get them, otherwise no.

Reply:
to speedup when cfstring is invaild or filled with zeros or ffffs. in theory none of those addresses should be available in the hashtable. but yeah we can skip that unnecessary check that will be matching in the 99.9% of cases